theano/gof/params_type.py

```diff
@@ -202,7 +270,7 @@ def __eq__(self, other):
     def __hash__(self):
         return hash((type(self),) + self.fields + self.types)
 
-    def generate_struct_name(self):
+    def __generate_struct_name(self):
```

Comment:
I was thinking this method should be private, as it should not be called outside this class (it is used to generate the C struct name, and this name is available via `self.name` after class is instantiated).

But, indeed, it may not be so useful.